test/var.cpp

```diff
@@ -114,8 +114,8 @@ TYPED_TEST(Var, DimCPPSmall) {
     vector<vector<TypeParam> > in;
     vector<vector<outType> > tests;
 
-    readTests<TypeParam, outType, double>(TEST_DIR "/var/var.data", numDims, in,
-                                          tests);
+    readTests<TypeParam, outType, float>(TEST_DIR "/var/var.data", numDims, in,
```

Comment:
Are these changes to reduce number of instantiations for `readTests` ?

Reply:
yes